scheduler/scheduler-examples/src/main/java/org/ow2/proactive/scheduler/examples/SQLPooledConnections.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * ProActive Parallel Suite(TM):
+ * The Open Source library for parallel and distributed
+ * Workflows & Scheduling, Orchestration, Cloud Automation
+ * and Big Data Analysis on Enterprise Grids & Clouds.
+ *
+ * Copyright (c) 2007 - 2017 ActiveEon
+ * Contact: contact@activeeon.com
+ *
+ * This library is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License
+ * as published by the Free Software Foundation: version 3 of
+ * the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * If needed, contact us to obtain a release under GPL Version 2 or 3
+ * or a different license than the AGPL.
+ */
+package org.ow2.proactive.scheduler.examples;
+
+import java.sql.Connection;
+import java.util.HashMap;
+import java.util.Map;
+
+
+/**
+ * @author ActiveEon Team
+ * @since 20/12/2018
+ */
+public class SQLPooledConnections {
+
+    private static Map<String, Connection> SQLPooledConnectionsMap = new HashMap<>();
```

Comment:
you dont use any kind of synchronization.
I dont think HashMap is tread-safe. You can replace it with ConcurrentHashMap

Reply:
Thanks @Mykhailenko for the review. You are right. I will replace it with ConcurrentHashMap.